keras_cv/models/stable_diffusion/ptp_utils.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility methods used to implement Prompt-to-Prompt paper in TensorFlow.
+
+Reference:
+
+- "Prompt-to-Prompt Image Editing with Cross-Attention Control."
+  Amir Hertz, Ron Mokady, Jay Tenenbaum, Kfir Aberman, Yael Pritch, Daniel Cohen-Or.
+  https://arxiv.org/abs/2208.01626
+
+Credits:
+
+- Official implementation of the paper: https://github.com/google/prompt-to-prompt
+"""
+
+from typing import Tuple
+
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+import keras_cv.models.stable_diffusion.seq_aligner as seq_aligner
+from keras_cv.models.stable_diffusion.diffusion_model import td_dot
+
+
+def rename_cross_attention_layers(diff_model: tf.keras.Model):
```

Comment:
nit: let's be consistent about naming attn vs attention.

I personally prefer the more verbose "attention" but am happy either way, as long as it's consistent.

Reply:
Renamed it to `rename_cross_attn_layers` to maintain consistency since I use attn everywhere basically. But I do prefer "attention" actually.

Reply:
End up renaming it to `rename_cross_attention_layers`